### ISSUE TITLE
Installing RStudio, get “TypeError” when opening the application, followed by XCB error: 3 (BadWindow)

### ISSUE THREAD

**Issue by axiomtutor** [rstudio/rstudio#4426]
I'm running a Linux Mint machine with version 18.03. I've previously had RStudio installed and working, however when I opened the application recently, it would open a blank window with the menu, but it was not able to drop down any of the menu options and was non-responsive in every other way too. So I ran sudo apt purge rstudio and then also opened the Anaconda navigator and made sure the application was un-installed from there too. So as far as I can tell, all old versions of the application are gone from my computer. I tried using the package manager to install again, but the application still failed on opening. So I went to the RStudio site and downloaded the latest installer, ran it, it seemed to install but again when it opens it crashes. Only this time I get the error message in the terminal:

```
QApplication: invalid style override passed, ignoring it.
"TypeError: undefined is not an object (evaluating 'window.desktopHooks.notifyRCrashed')"
QXcbConnection: XCB error: 3 (BadWindow), sequence: 883, resource id: 19954298, major code: 40 (TranslateCoords), minor code: 0
```

In other related issues that I see on here there are mentions of running an install-dependencies script but I don't find the file path that is referenced there.  I didn't download any ZIP file but I figured the installer that I downloaded would have created any necessary directories--or am I mistaken?  I'm not an expert user of Linux systems so I'm not fully aware of how binaries, source files, installers, etc. work.  But since this .deb file is the main download file and the only instructions I see here is to make sure R is installed: https://www.rstudio.com/products/rstudio/download/#download  Then I would assume I don't need to do anything other than install using `sudo dpkg -i ...`

[Edit: I also went into a terminal session for R and updated all packages, so hopefully R itself is entirely up to date.]

**Comment by pascaloettli** [comment]
You don't say which version of RStudio you are using, despite being asked to when opening an issue:
### System details

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       : 

According to the link, you are trying to use v.1.2. Also, this message is visible when opening an issue:

> RStudio v1.2 is currently available as a preview release, from https://www.rstudio.com/products/rstudio/download/preview/. If you are reporting a bug with RStudio v1.1, it would be greatly appreciated if you could check if the issue also reproduces in the preview release before reporting an issue.

Do you have the same problem with the preview version of RStudio?

**Comment by ronblum** [comment]
@axiomtutor Following from @pascaloettli observation: by downloading the latest installer, do you mean 1.1.463 (which is the one available at the link you provided), or the 1.2 preview (at the link @pascaloettli provided)?

**Comment by ronblum** [comment]
Closing due to inactivity.
